src/components/PaginatedTable/types.ts

```diff
@@ -42,6 +43,13 @@ export interface PaginatedTableData<T> {
     found: number;
 }
 
+export type StorageNodesPaginatedTableData = PaginatedTableData<PreparedStorageNode> & {
```

Comment:
It's better to place it somewhere near `useTableCSSVariables` in `Storage`, since it isn't a type, that generally related to `PaginatedTable` as component